content/pt-br/docs/concepts/configuration/windows-resource-management.md

```diff
@@ -0,0 +1,75 @@
+---
+reviewers:
+- jayunit100
+- jsturtevant
+- marosset
+- perithompson
+title: Gerenciamento de recursos para nós do Windows
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+Esta página descreve as diferenças em como os recursos são gerenciados entre o Linux e o Windows.
+
+<!-- body -->
+
+Em nós do Linux, {{< glossary_tooltip text="cgroups" term_id="cgroup" >}} são usados como um limite de pod para controle de recursos. 
+Os contêineres são criados dentro desse limite para o isolamento de rede, processo e sistema de arquivos. 
+As APIs Linux cgroup podem ser usadas para coletar estatísticas de uso de CPU, E/S e memória.
```

Comment:
Some suggestions in this paragraph:
* Use "nós Linux" instead of "nós do Linux".
* How do you feel about using "divisa" instead of "limite" as translation for boundary?
* How do you feel about rephrasing the first sentence as "são usados como uma divisa para o controle de recursos em Pods"?
* In the last sentence, perhaps "As APIs de cgroup do Linux..." would sound better.